.ci/gitlab/.gitlab-ci.yml

```diff
@@ -46,14 +46,10 @@ default:
         SPACK_COPY_BUILDCACHE: "${PROTECTED_MIRROR_PUSH_DOMAIN}/${CI_COMMIT_REF_NAME}"
         SPACK_REQUIRE_SIGNING: "True"
         OIDC_TOKEN_AUDIENCE: "protected_binary_mirror"
-    - if: $CI_COMMIT_TAG =~ /^develop-[\d]{4}-[\d]{2}-[\d]{2}$/ || $CI_COMMIT_TAG =~ /^v.*/
-      # Pipelines on tags (release or dev snapshots) only copy binaries from one mirror to another
+    - if: $CI_COMMIT_TAG =~ /^v.*/ || $CI_COMMIT_TAG =~ /^develop-[\d]{4}-[\d]{2}-[\d]{2}$/
       when: always
       variables:
-        SPACK_PIPELINE_TYPE: "spack_copy_only"
-        SPACK_COPY_BUILDCACHE: "${PROTECTED_MIRROR_PUSH_DOMAIN}/${CI_COMMIT_REF_NAME}"
-        PIPELINE_MIRROR_TEMPLATE: "copy-only-protected-mirrors.yaml.in"
-        OIDC_TOKEN_AUDIENCE: "protected_binary_mirror"
+        NOOP_JOB: 1
```

Comment:
this should probably default to 0, right?

Reply:
@kwryankrattiger mentioning you to notify you of the review (with a request to change)

Reply:
The checks that use this only check if it is set, it isn't defined in any of the other pipeline cases.